### PR TITLE
TMS 358 Taulukossa on liian pieni ja väärä fontti

### PR DESCRIPTION
**Pull request by tim0haapala**
[https://geniemdev.atlassian.net/browse/TMS-10](https://geniemdev.atlassian.net/browse/TMS-10)

## Description
Font fix for single artist's page's table element.

## Motivation and Context

The text styles were different than in Figma.

[https://geniemdev.atlassian.net/browse/TMS-358](https://geniemdev.atlassian.net/browse/TMS-358)

## How Has This Been Tested?

Tested visually and by comparing Figma to staging sites

Tested with composer lint-all 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly (CHANGELOG).
- [x] I have read the [CONTRIBUTING] document.

### CODE REVIEW

File: CHANGELOG.MD [hunk 1]
```diff
@@ -24,6 +36,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - TMS-336: Override pill color scheme #45
 - TMS-335: Image banner block button colour change #44
+>>>>>>> TMS-348
```
Comment (visahakkarainen) [inline CHANGELOG.MD:39]: This should be removed.

**Review by visahakkarainen — APPROVED** [review]
Check comment(s).

**Comment by tim0haapala** [comment]
Stuff from stage can be found from this PR, so closing this and making a new one prperly

**Comment by tim0haapala** [comment]
Stuff from stage can be found from this PR, so closing this and making a new one properly

**Pull request closed without merging**